wallet.go

```diff
@@ -679,7 +680,8 @@ func (this *WalletData) Load(path string) error {
 	if err != nil {
 		return err
 	}
-	return json.Unmarshal(msh, this)
+	body := bytes.TrimPrefix(msh, []byte("\xef\xbb\xbf")) // Or []byte{239, 187, 191}
```

Comment:
？？？
suggest to trim all unread ascii ?

Reply:
the code in this file is duplicated with `ontology`, need refactor.